src/govuk/all.mjs

```diff
@@ -58,6 +59,13 @@ function initAll (config) {
     new ErrorSummary($errorSummary, config.errorSummary).init()
   }
 
+  var $exitThisPageButtons = $scope.querySelectorAll('[data-module="govuk-exit-this-page"]')
+  if ($exitThisPageButtons.length > 0) {
```

Comment:
Shall we ditch the length check? For the other components we go straight to `nodeListForEach()`

```suggestion
```